gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/dto/LogMessage.java

```diff
@@ -51,13 +55,11 @@
 	}
 	
 	String process_id;
-	String parent_process_id;
-
 	long timestamp;
 	String log_message;
 	LogLevel log_level;
 	ProcessStatus process_status;
-	Boolean storeToDB = true;
+	Boolean store_to_db = true;
 	
 	//I would change timestamp to a long, log level and process status to enums. and probably process id to a numeric.  and storeToDB should be store_to_db for consistency
```

Comment:
Note the comment...I would do that...as this is java the names should be lowerUpperCase formatted.

Reply:
Done